experiment_handling.py

```diff
@@ -119,12 +119,20 @@ def resave_data(save_path, parameter_combinations, index, eva, name,
         The number of samples to use [Default: None (means: Use all samples)]
     badmisskey : string (optional)
         key that misses in the result dictionaries for badruns (Default: None)
+    save_path : string
```

Comment:
the documentation of the input parameters is in order of inputs in the function definition again with the save_path last.